### DISCUSSION TITLE
The discmarket should be editorial Editing option

### DISCUSSION THREAD

**Discussion by saber716rus** [rhinstaller/anaconda#3772] [Ideas]
Hello. When markup, there is no possibility to edit the installation option. For example: I have SSD and hdd, for example, I want to use compression, and add autodefrag, through the installation options * * express=zstd* and *autodefrag** which are credited in fstab.

**Comment by poncovka** [comment]
Hi, you should be able to specify the additional options using a kickstart file and `--fsoptions`. See:
https://pykickstart.readthedocs.io/en/latest/commands.html?highlight=--fsoptions#part-or-partition

**Reply by saber716rus** [reply]
That is, it turns out that this feature is already embedded in this distribution installer, then I will ask this question to the maintainers of the distribution.

**Comment by saber716rus** [comment]
> Hi, you should be able to specify the additional options using a kickstart file and `--fsoptions`. See:
> https://pykickstart.readthedocs.io/en/latest/commands.html?highlight=--fsoptions#part-or-partition

It only works for tastarte, but not for gauking. Detail I described for fucking here:
https://github.com/storaged-project/blivet-gui/issues/323
